app/Http/Controllers/V2MiscController.php

```diff
@@ -0,0 +1,235 @@
+                    ->with('route', route('forum.store.misc'))
+                    ->with('fields', collect()
+                        ->push(component('FormRadio')
+                            ->with('name', 'type')
+                            ->with('value', 'forum')
+                            ->with('options', collect(['misc'])
+                                ->map(function ($type) {
+                                    return collect()
+                                        ->put('id', $type)
+                                        ->put('name', trans("content.$type.index.title"));
+                                })
+                            )
+                        )
+                        ->push(component('FormTextfield')
+                            ->is('large')
+                            ->with('title', trans('content.forum.edit.field.title.title'))
+                            ->with('name', 'title')
+                            ->with('value', old('title'))
+                        )
+                        ->push(component('FormTextarea')
+                            ->with('title', trans('content.forum.edit.field.body.title'))
+                            ->with('name', 'body')
+                            ->with('value', old('title'))
+                            ->with('rows', 20)
+                        )
+                        ->push(component('FormButton')
+                            ->is('hidden')
+                            ->with('title', trans('content.create.submit.title'))
+                        )
+                        ->push(component('FormButtonProcess')
+                            ->with('id', 'ForumCreateForm')
+                            ->with('title', trans('content.create.submit.title'))
+                            ->with('processingtitle', trans('content.create.submitting.title'))
+                        )
+                    )
+                )
+            )
+
+            ->with('sidebar', collect()
+                ->push(component('Block')
+                    ->is('gray')
+                    ->with('content', collect()
+                        ->push(component('Title')
+                            ->is('smaller')
+                            ->is('red')
+                            ->with('title', trans('content.edit.notes.heading'))
+                            ->with('route', route('forum.index'))
+                        )
+                        ->push(component('Body')
+                            ->with('body', trans('content.edit.notes.body'))
+                        )
+                    ))
+            )
+
+            ->with('footer', region('Footer'))
+
+            ->render();
+    }
+
+    public function store(Request $request, Content $content = null)
+    {
+        $this->validate($request, $this->rules);
+
+        return $content->id ? $this->update($request, $content) : $this->newContent($request, $request->type);
```

Comment:
Lets just have storing stuff here, do not try store and update in same go. copy the ```newContent``` stuff here